app/conf/Configuration.scala

```diff
@@ -202,8 +202,6 @@ class ApplicationConfiguration(val playConfiguration: PlayConfiguration, val isP
 
     val contentApiLiveHost: String = getMandatoryString("content.api.host")
     def contentApiDraftHost: String = getMandatoryString("content.api.draft.iam-host")
-
-    val editionsPrefillHost: String = getMandatoryString("content.api.editions.prefillHost")
     lazy val editionsKey: String = getMandatoryString("content.api.editions.apiKey")
```

Comment:
This can go too I think

Reply:
i tried to remove the apiKey and it was not working locally without it

Reply:
Regarding JSONB question, i think it will make sense to go with JSONB if we will need to include more params for the query object in the future, for that one the model is quite simple so i don't think we paid a lot of attention to decide if we should go with JSONB or additional field

One of the advantage of not using JSONB is not having issue with serialisation of past state and not being forced to use Option[] fields as default for non optional params ;)

We can go with JSONB if that will be better but that will require some changes

Reply:
Huh. What was it that broke?

Reply:
Had a chat IRL - it seems like the CAPI Scala client requires a key to be provided so we'll leave this piece of config in for now.